_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,155 @@
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately.
+
+By default, you should strive to keep as many of your working documents in the open section of your workspace.
+But when needed, you can save sensitive information in the private folders and keep working in the open for the rest.
+
+### Manage leaks
+
+In software development, we expect that there will be a security issue at some point, while we continue to strive to minimize that eventuality.
+
+By putting in place a remediation process, we ensure that even if there was to be a leak, appropriate mesures are in place to address it as quickly as possible.
+And we make sure that everyone knows how to run through the process, if it's not fully automated.
+
+### Open up at inception
+
+Part of working in the open is accepting that we don't know everything and that others may be able to jump in and help at any time.
+
+When one goes into draft mode, it's easy to think that we can do better before getting feedback, so the initial reaction most people have is to wait until "it's good enough".
+
+If you are worried to be slowed down by feedback on a document that is barely an outline, you probably can comfort yourself with the thought that, in practice, chances are that contributions would not come before your product or document is actually mature enough!
+
+Plus, waiting too long may put you in the position where you have not set your mind into an open mode, continuously expecting someone to read it.
+That may lead you to add a review phase before opening up to ensure that no sensitive material has been included, instead of working from the start with that mindset.
+
+### Be humble and open minded
+
+Working in the open may seem like putting yourself out there, subject to criticism and judgement of others.
+
+But remember that no one knows everything.
+Opening up is an opportunity for you and your team to have others contribute to your work and to complement what you already know if they are interested in it.
+
+We often seek other teams feedback in an structured manner and that official process can still occur while receiving live feedback in between "versions" of the documents you produce.
+
+### Manage the community
+
+An argument sometimes used for avoiding working in the open is that managing the community may end up being more work than what you get back.
+Or that receiving external contributions before a certain version is mature enough would slow you down.
+
+Based on our experience, small and continuous feedback makes the process much less cognitively intensive since contributions usually are smaller and can be more easily integrated in the workflow.
+And by continuously reviewing and integrating them, it helps minimize the disruption of the entire effort.
+
+```markdown
```

Comment:
This is difficult. In a science-based community, document versioning is used to also communicate "where we are in the process". I.e. drafting? About to Publish? Published and official version?

In the software world, it's a CI/CD pipeline. But in an information/document world, there would need to have some soft of gating process. Otherwise it's everyone just contributing while not allowing a previous person comment on a future person's change to their paragraph.

The review process (e.g. 5 business days) is also meant to communicate "this is now your time to contribute. If you miss that opportunity, don't complain. We're moving forward". 

I've seen too often an attempt to collaborate that lead to an "almost ready to publish" when teams start raising their hands saying they were not involved while we left plenty of time to have them contribute. It's frustrating. And in this scenario we cannot continue because we need everyone's input before a senior leader is comfortable to continue.

I think the example below is more around allowing "informal review" before starting a formal peer review process. So that you get early feedback and can much better produce a document/product of quality. This reduces waste later on when peer review may shred your document because they don't see themselves in it.

Reply:
Yes, the idea is to have continuous contributions throughout the entire lifecycle of whichever document you're working on. Doesn't mean there isn't an official review at some point.

But that review may actually be a lot "faster" or leaner if folks have known it's been in the works. Maybe not either. But one doesn't exclude the other.